### ISSUE TITLE
compiling problem

### ISSUE THREAD

**Issue by YuriGen2423** [mlogjs/mlogjs#128]
most people are not gonna use the if statement like this but...
basically:
```javascript
let num1 = 100
let num2 = 150

if (num1 < 150) { }
else {
    printFlush()
}

// expected output is to be nothing on message1
```
output:
```asm
set num1:1:4 100
set num2:2:4 150
op greaterThan &t0 num1:1:4 num2:2:4
jump 5 always
print "num1 is less or equal than num2"
printflush message1
end
```
i think the jump instruction shouldn't be "always"
but instead a not equal

**Comment by JeanJPNM** [comment]
The fix is available in the online editor.

**Comment by JeanJPNM** [comment]
The fix is also available in the version 0.4.10 of the compiler
